### PR TITLE
DB/SQL item_mods relic+2

### PR DESCRIPTION
**Pull request by wannabc5**
Complete set of item_mods Relic+2

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -1404,23 +1404,698 @@ INSERT INTO `item_mods` VALUES (10650, 9, 8);
...
+INSERT INTO `item_mods` VALUES (10678, 391, 8); -- Charm+8
+
+-- ------------------------------------------
+-- Bard's Jstcorps +2
+-- ------------------------------------------
+INSERT INTO `item_mods` VALUES (10679, 1, 59); -- DEF:59
+INSERT INTO `item_mods` VALUES (10679, 2, 27); -- HP+27
+INSERT INTO `item_mods` VALUES (10679, 25, 25); -- ACC+25
+INSERT INTO `item_mods` VALUES (10679, 23, 25); -- Attack+25
+INSERT INTO `item_mods` VALUES (10679, 8, 10); -- Brd. Jstcorps +2  "Military Parade" unknown added base stats
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: You got bad code from Legion servers very old and often wrong SQL lines from that forum post. There was a reason they never included their mod sin a pull request yet, and Deathpod's substitute modifiers was a major one.

Comment (teschnei) [inline sql/item_mods.sql]: spoiler alert: military parade is easily done with a latent

Comment (TeoTwawki) [inline sql/item_mods.sql]: double spoiler: That mod ID used on line 1553 is actually scavenge effect right now, not avatar mab.

**Comment by wannabc5** [comment]
Thanks for the input. I'll make sure I triple check the mods. Thanks for the feedback.

**Pull request merged**